tests/test_lsst_efd_client.py

```diff
@@ -240,6 +253,16 @@ async def test_time_series(efd_client, start_stop):
     assert numpy.all(t == 0.)
     # But the queries should be different
     assert not efd_client.query_history[-2] == efd_client.query_history[-1]
+    # Test indexed query
+    df1 = await efd_client.select_time_series('lsst.sal.barSubSys.test', ['eggs', 'ham'],
+                                              start_stop[0], start_stop[1], index=2)
+    assert len(df1) == 100
+    assert numpy.all(df1["eggs"] == 10)
+    # Old index should return an empty dataframe
+    df1 = await efd_client.select_time_series('lsst.sal.barSubSys.test', ['eggs', 'ham'],
+                                              start_stop[0], start_stop[1], index=2,
+                                              use_old_csc_indexing=True)
+    assert len(df1) == 0
```

Comment:
I assume this is because there are no entries with the old index name.  Is it too hard to add a few entries to test the old style indexing?

Reply:
Sounds reasonable. Let me see what I can do.

Reply:
I've added the request tests if you want to take a look.